grumphp.yaml

```diff
@@ -1,7 +1,6 @@
-parameters:
-    git_dir: .
-    bin_dir: "./vendor/bin"
+grumphp:
```

Comment:
on new grum there is simply `phpcsfixer` task, isn't it? `phpcsfixer2` works?
You can also upgrade phpcsfixer too.